product_extended_segmentation/model/template.py

```diff
@@ -298,13 +295,13 @@ def ensure_change_price_log_messages(self, vals):
             body += ""
         else:
             subject += 'I cowardly did not update cost, Standard new\n'
-            body += "Price is {comp_th:.2f}% less than old price\n"\
-                "new {new} old {old} \n"\
-                "(current max allowed is {max_th})\n"\
-                .format(old=vals['current_price'],
-                        new=new_price['standard_price'],
-                        comp_th=vals['computed_th'],
-                        max_th=vals['current_bottom_th'])
+            body += "Price is %(comp_th)s.2f%% less than old price\n"\
```

Comment:
The result of `%(comp_th)s.2f%%` vs `{comp_th:.2f}%` using a floating number is very different.
Could you check it?